middlewares/authorization.js

```diff
@@ -0,0 +1,14 @@
+const { isMember } = require('../models/members')
+module.exports = (req, res, next) => {
+  try {
+    const { username } = req.userData
+    const isUserMember = isMember(username)
```

Comment:
Stocks should not have been created by members, only from authorized users (currently just `ankush`)